[BUG] detect-secrets AzDO snippet contains errors

**Describe the bug**

The recipe for detecting secrets in AzDO, "(Detecting Secrets in your Azure DevOps Pipeline with YELP detect-secrets)[https://github.com/microsoft/code-with-engineering-playbook/blob/main/docs/continuous-integration/dev-sec-ops/secret-management/recipes/detect-secrets-ado.md]", contains few minor errors.

**To Reproduce**
Steps to reproduce the behavior:

1. Copy the Ubuntu version of the YAML and create a pipeline in AzDO
1. Run the pipeline
1. See that `FETCH_HEAD` file is detected to contain `hashed_secret` so the pipeline fails

After the change to exclude the `FETCH_HEAD` file:

1. Run the pipeline
2. See the error `##[error]Secrets were detected in code. 0 file(s) affected.`

--- a/docs/continuous-integration/dev-sec-ops/secret-management/recipes/detect-secrets-ado.md
+++ b/docs/continuous-integration/dev-sec-ops/secret-management/recipes/detect-secrets-ado.md
@@ -42,7 +42,7 @@ jobs:
 
       - bash: |
           detect-secrets --version
-          detect-secrets scan --all-files --force-use-all-plugins > $(Pipeline.Workspace)/detect-secrets.json
+          detect-secrets scan --all-files --force-use-all-plugins --exclude-files FETCH_HEAD > $(Pipeline.Workspace)/detect-secrets.json
         displayName: "Run detect-secrets tool"
 
       - task: PublishPipelineArtifact@1
@@ -58,7 +58,7 @@ jobs:
 
           count=$(echo "${dsjson}" | jq -c -r '.results | length')
 
-          if [ count > 0 ]; then
+          if [ $count -gt 0 ]; then
             msg="Secrets were detected in code. ${count} file(s) affected."
             echo "##vso[task.logissue type=error]${msg}"
             echo "##vso[task.complete result=Failed;]${msg}."

--- a/docs/developer-experience/recipes/client-app-inner-loop.md
+++ b/docs/developer-experience/recipes/client-app-inner-loop.md
@@ -6,7 +6,7 @@ don't always fully align. For a high velocity inner dev loop, client app
 development must be decoupled from the backend services while still
 allowing the app to "invoke" the services for local testing.
 
-# Options
+## Options
 
 Several options exist to decouple client app development from the
 backend services. The options range from embedding mock implementation
@@ -31,7 +31,7 @@ A simple service implementation can return a static response. For
 RESTful services, the JSON responses for the stubs can be stored as
 application resources or simply as static strings.
 
-```
+```cs
 public Task\<UserProfile> GetUserAsync(long userId, CancellationToken
 cancellationToken)
 {
@@ -62,7 +62,7 @@ it easy to switch between mock services and real service client
 implementations. Since both implement the same interface,
 implementations can be registered with the Unity container.
 
-```
+```cs
 public static void Bootstrap(IUnityContainer container)
 {
 
@@ -79,11 +79,9 @@ public static void Bootstrap(IUnityContainer container)
 
 The code consuming the interfaces will not notice the difference.
 
-```
+```cs
 public class UserPageModel
-
 {
-
     private readonly IUserServiceClient userServiceClient;
 
     public UserPageModel(IUserServiceClient userServiceClient)
@@ -144,8 +142,7 @@ fake json server API Service for prototyping and
 testing.](https://www.jsonserver.io/)) or Postman. All these services
 would respond with predetermined and configured JSON messages.
 
-# How to decide
-
+## How to decide
 
 || Pros| Cons| Example when developing for: | Example When not to Use  |
 |------------|-------------|-------------|---------------|------------|

